### DISCUSSION TITLE
Filter events via URL

### DISCUSSION THREAD

**Discussion by chrzimmer** [liedekef/events-made-easy#348] [Q&A]
Hello, i hope my question is not too stupid :)

I would like to call my events page in that form, that i can pass i.e. the event type as filter.
Even it may be nonsens, i think this should look like https,//example.com/events&type=markets

Is there something like that ? Could one show me an example ?

Thanks in advance and cheers to all 
Chris

**Comment by liedekef** [comment]
Combine the eme_events shortcode with the eme_filterform shortcode, that way you can see which fields can be given (check the POST result). All the fields supported by eme_filterform can be given as url-params too, so for example "eme_cat_filter" is a supported url-param. "type" is unknown to EME (events are not posts to begin with).

**Reply by chrzimmer** [reply]
Dear Franky, that you very much for this instant replay. To be honest, I am too  unexperienced with this topic. Could you guide me with an example ? My category is "Market".
I tried https://example.com/events&&eme_cat_filter="category=Market"  
But this fails :-/

Thanks again

**Reply by liedekef** [reply]
You have double "&", not ok. https://example.com/events&eme_cat_filter=Market  should be ok (if the EME category Market exists).

**Reply by chrzimmer** [reply]
Again I appreciate your help ! Unfortunately i get with the line above a 404 error (the double "&" was by accident). I also tried it with the category number, but also receive a 404. Like:
[https://dahmeland-nachrichten.de/veranstaltungen&eme_cat_filter=Feste](https://dahmeland-nachrichten.de/veranstaltungen&eme_cat_filter=Feste)
[https://dahmeland-nachrichten.de/veranstaltungen&eme_cat_filter=6](https://dahmeland-nachrichten.de/veranstaltungen&eme_cat_filter=6)

**Reply by liedekef** [reply]
I gave an example based on your example, but on your real site it should be something like this:

https://dahmeland-nachrichten.de/veranstaltungen/?eme_cat_filter=3

But currently it only works with the category ids, not the name/slug
Edit: the next version will also support the category name/slug as filter value
